eslint.config.mjs

```diff
@@ -3,37 +3,39 @@ import pluginJs from "@eslint/js";
 import tseslint from "typescript-eslint";
 import pluginReactConfig from "eslint-plugin-react/configs/recommended.js";
 
-
 export default [
-  pluginJs.configs.recommended,
-  ...tseslint.configs.recommended,
-  pluginReactConfig,
-  {
-    languageOptions: {
-      globals: {
-        ...globals.browser,
-        ...globals.jest,
-        browser: true,
-        module: true
-      }
+    {
+        ignores: [".next", "node_modules"]
     },
-    rules: {
-      indent: [
-          "error",
-          4
-        ],
-        "linebreak-style": [
-          "error",
-          "unix"
-        ],
-        quotes: [
-          "error",
-          "double"
-        ],
-        semi: [
-          "error",
-          "always"
-        ]
-    }
```

Comment:
Wait, I thought you linked these, why are we getting rid of them?

Reply:
It's deprecated. I plan to implement it correctly in a follow up PR:
https://eslint.org/docs/latest/rules/semi